CMakeLists.txt

```
@@ -108,3 +108,61 @@ install(TARGETS D1GraphicsTool
 if(QT_VERSION_MAJOR EQUAL 6)
     qt_finalize_executable(D1GraphicsTool)
 endif()
+
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+  set(CPACK ON)
+endif()
+
+# RELEASE_OPTION(CPACK "Configure CPack")
+if(CPACK)
```

I think it would also be fine if we do this always:
```suggestion
if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
```

---

Done.